test/test_text_recoginse_emotion.py

```diff
@@ -0,0 +1,52 @@
+# coding: utf-8
+
+"""
+    DeepAffects
+
+    OpenAPI spec version: v1
+"""
+
+
+from __future__ import absolute_import
+
+import os
+import sys
+import unittest
+
+import deepaffects
+from deepaffects import Audio
+from deepaffects.rest import ApiException
+from deepaffects.apis.diarize_api_v2 import DiarizeApiV2
+DIR = os.path.dirname(os.path.realpath(__file__))
+import uuid
+
+
+class TestTextEmootion(unittest.TestCase):
+    """ Text Emotion unit test stubs """
+
+    def setUp(self):
+        deepaffects.configuration.api_key['apikey'] = os.environ['DEEPAFFECTS_API_KEY']
+        self.webhook_url = os.environ["DEEPAFFECTS_API_WEBHOOK"]
+        self.api = deepaffects.apis.emotion_api.EmotionApi()
+        self.request_id = str(uuid.uuid4())
+
+    def tearDown(self):
+        pass
+
+    def test_async_diarize_audio(self):
```

Comment:
you aren't testing diarize audio here.

Reply:
Add more tests to test the cases where the request is malformed. Also check for all the expected keys in output